tests/test_util.py

```diff
@@ -33,20 +33,27 @@ def test_lookups(self):
         eq_("en", c.three_to_two['eng'])
         eq_(["English"], c.english_names['en'])
         eq_(["English"], c.english_names['eng'])
+        eq_(["English"], c.native_names['en'])
+        eq_(["English"], c.native_names['eng'])
 
         eq_("spa", c.two_to_three['es'])
         eq_("es", c.three_to_two['spa'])
         eq_(["Spanish", "Castilian"], c.english_names['es'])
         eq_(["Spanish", "Castilian"], c.english_names['spa'])
+        eq_([unicode("español", "utf-8"), "castellano"], c.native_names['es'])
+        eq_([unicode("español", "utf-8"), "castellano"], c.native_names['spa'])
 
         eq_("chi", c.two_to_three['zh'])
         eq_("zh", c.three_to_two['chi'])
         eq_(["Chinese"], c.english_names['zh'])
         eq_(["Chinese"], c.english_names['chi'])
+        eq_([unicode("äæ (Zhōngwén)", "utf-8"), unicode("æè", "utf-8"), unicode("æè", "utf-8")], c.native_names['zh'])
+        eq_([unicode("äæ (Zhōngwén)", "utf-8"), unicode("æè", "utf-8"), unicode("æè", "utf-8")], c.native_names['chi'])
```

Comment:
Maybe I should get rid of the full list and we can get individual translations for the languages we support.